source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -120,6 +120,9 @@ def _align_fallback_direct_path_datetime(fallback_direct_path, fallback_extremit
         for s in journey.sections:
             s.begin_date_time += delta
             s.end_date_time += delta
+            if s.base_begin_date_time != None and s.base_end_date_time:
```

Comment:
The culprit for the diff, during the test.
Base date time needs to be drift like classic date time